command/issue.go

```diff
@@ -61,6 +61,14 @@ var issueCreateCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create a new issue",
 	RunE:  issueCreate,
+	Example: `$ gh issue create
+$ gh issue create -t "Issue With Title And Body" -b "This is the issue body"
+$ gh issue create -t OpenAWebBrowserToCreateThisIssue --web
+$ gh issue create -t IssueWithLabels -b IssueBody -l label1,label2
+$ gh issue create -t IssueWithAssignees -b IssueBody -a user1Login,user2Login
+$ gh issue create -t IssueWithProjects -b IssueBody -p cli/1,cli/2
+$ gh issue create -t IssueWithMilestone -b IssueBody -m someMilestone
+`,
```

Comment:
Instead of documenting all the flags I think we only need to document the ones that can cause confusion. I also used the expanded flag names to make the examples easier to read.

```suggestion
	Example: `$ gh issue create --title "I found a bug" --body "Nothing works"
$ gh issue create --label label1,label2
$ gh issue create --author user1Login,user2Login
$ gh issue create --project cli/1,cli/2`,
```

Reply:
Agreed, and I would also add
* maybe it would be a good idea to also demonstrate the alternate syntax for specifying multiple labels: `--label label1 --label label2`
* `--author user1Login,user2Login` - this is not supported. we cannot query by multiple authors
* `--project cli/1,cli/2` - I find the slashes here confusing, since it looks like it might be some kind of special syntax, but is in fact just the project name. I would suggest to only show an example of setting a single project by name, but maybe using double quotes since projects usually have multi-word names